experimental/BasisLieHighestWeight/src/BasisLieHighestWeight.jl

```diff
@@ -26,6 +26,7 @@ import Base: length
 # - the list of Minkowski gens contains too many elements, only include those that give us something new
```

Comment:
further up in this file you probably need to add
```julia
import Oscar: base_lie_algebra
import Oscar: character
```
so that the methods you add for these get added to the already existing function, and you don't create a new, separate (but same-named) function. This should help with your `LieAlgebraModule.character` problem as well.